src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinConversionExtensions.cs

```diff
@@ -30,12 +30,12 @@ internal static class ZipkinConversionExtensions
 
         private static readonly Dictionary<string, int> RemoteEndpointServiceNameKeyResolutionDictionary = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase)
         {
-            ["net.peer.name"] = 0, // RemoteEndpoint.ServiceName primary.
-            ["peer.service"] = 0, // RemoteEndpoint.ServiceName primary.
-            ["peer.hostname"] = 1, // RemoteEndpoint.ServiceName alternative.
-            ["peer.address"] = 1, // RemoteEndpoint.ServiceName alternative.
-            ["http.host"] = 2, // RemoteEndpoint.ServiceName for Http.
-            ["db.instance"] = 2, // RemoteEndpoint.ServiceName for Redis.
+            [SpanAttributeConstants.PeerServiceKey] = 0, // RemoteEndpoint.ServiceName primary.
```

Comment:
I made Zipkin & Jaeger consistent, but I wasn't sure what should have top priority: "net.peer.name" or "peer.service." I went with "peer.service" because that's what it is called in the Jaeger protocol, but OT spec calls out "net.peer.name" specifically. I have a feeling it will change a bunch.

Reply:
This isn't related to the SqlCollector - please can you move these changes into a separate PR where we can discuss what the priority should be too.

Reply:
@MikeGoldsmith Would you be open to leaving on here and then I'll open an issue ticket for further discussion? Just trying to save a little time.

Reply:
Yeah, we can leave it here.